### PR TITLE
Return a Quantity object for scalar results

### PR DESCRIPTION
**Pull request by mulcahyk**
For external BONNIE-260 updated AssessmentPerformed to return a Quantity object for the result when the result is of type Scalar.

### CODE REVIEW

File: app/assets/javascripts/datatypes/assessment.js.coffee [hunk 1]
```diff
@@ -61,11 +61,11 @@ class CQL_QDM.AssessmentPerformed extends CQL_QDM.QDMDatatype
         new cql.Code(code, code_system)
       else if @_result.units == "UnixTime" # A PhysicalQuantity with unit UnixTime is a TimeStamp
         CQL_QDM.Helpers.convertDateTime(@_result.scalar)
-      else
-        parseInt(@_result.scalar)
+      else if @_result.scalar?.length && @_result.units?.length
```
Comment (holmesie) [inline app/assets/javascripts/datatypes/assessment.js.coffee]: What is the `.length` used for here?

Comment (mayerm94) [inline app/assets/javascripts/datatypes/assessment.js.coffee]: It should probably just be **@_result.scalar?**

**Comment by mayerm94** [comment]
@c-monkey The reason why it "breaks a ton of other patients" is likely because the logic requires >259 days and the results in the patients are given in weeks. (ie 37 weeks) If you replace the result value with 37 * 7 = 259 days the fails will turn into passes

**Comment by c-monkey** [comment]
Good catch, @mayerm94. @mulcahyk and I had already determined this. I need to discuss this with the measure developers before we deploy it as it could be a big issue.

**Pull request merged**